manifest/armv7l/h/harfbuzz.filelist

```diff
@@ -54,6 +54,60 @@
 /usr/local/include/fontconfig/fcfreetype.h
 /usr/local/include/fontconfig/fcprivate.h
 /usr/local/include/fontconfig/fontconfig.h
+/usr/local/include/freetype2/freetype/config/ftconfig.h
+/usr/local/include/freetype2/freetype/config/ftheader.h
+/usr/local/include/freetype2/freetype/config/ftmodule.h
+/usr/local/include/freetype2/freetype/config/ftoption.h
+/usr/local/include/freetype2/freetype/config/ftstdlib.h
+/usr/local/include/freetype2/freetype/config/integer-types.h
+/usr/local/include/freetype2/freetype/config/mac-support.h
+/usr/local/include/freetype2/freetype/config/public-macros.h
+/usr/local/include/freetype2/freetype/freetype.h
+/usr/local/include/freetype2/freetype/ftadvanc.h
+/usr/local/include/freetype2/freetype/ftbbox.h
+/usr/local/include/freetype2/freetype/ftbdf.h
+/usr/local/include/freetype2/freetype/ftbitmap.h
+/usr/local/include/freetype2/freetype/ftbzip2.h
+/usr/local/include/freetype2/freetype/ftcache.h
+/usr/local/include/freetype2/freetype/ftchapters.h
+/usr/local/include/freetype2/freetype/ftcid.h
+/usr/local/include/freetype2/freetype/ftcolor.h
+/usr/local/include/freetype2/freetype/ftdriver.h
+/usr/local/include/freetype2/freetype/fterrdef.h
+/usr/local/include/freetype2/freetype/fterrors.h
+/usr/local/include/freetype2/freetype/ftfntfmt.h
+/usr/local/include/freetype2/freetype/ftgasp.h
+/usr/local/include/freetype2/freetype/ftglyph.h
+/usr/local/include/freetype2/freetype/ftgxval.h
+/usr/local/include/freetype2/freetype/ftgzip.h
+/usr/local/include/freetype2/freetype/ftimage.h
+/usr/local/include/freetype2/freetype/ftincrem.h
+/usr/local/include/freetype2/freetype/ftlcdfil.h
+/usr/local/include/freetype2/freetype/ftlist.h
+/usr/local/include/freetype2/freetype/ftlzw.h
+/usr/local/include/freetype2/freetype/ftmac.h
+/usr/local/include/freetype2/freetype/ftmm.h
+/usr/local/include/freetype2/freetype/ftmodapi.h
+/usr/local/include/freetype2/freetype/ftmoderr.h
+/usr/local/include/freetype2/freetype/ftotval.h
+/usr/local/include/freetype2/freetype/ftoutln.h
+/usr/local/include/freetype2/freetype/ftparams.h
+/usr/local/include/freetype2/freetype/ftpfr.h
+/usr/local/include/freetype2/freetype/ftrender.h
+/usr/local/include/freetype2/freetype/ftsizes.h
+/usr/local/include/freetype2/freetype/ftsnames.h
+/usr/local/include/freetype2/freetype/ftstroke.h
+/usr/local/include/freetype2/freetype/ftsynth.h
+/usr/local/include/freetype2/freetype/ftsystem.h
+/usr/local/include/freetype2/freetype/fttrigon.h
+/usr/local/include/freetype2/freetype/fttypes.h
+/usr/local/include/freetype2/freetype/ftwinfnt.h
+/usr/local/include/freetype2/freetype/otsvg.h
+/usr/local/include/freetype2/freetype/t1tables.h
+/usr/local/include/freetype2/freetype/ttnameid.h
+/usr/local/include/freetype2/freetype/tttables.h
+/usr/local/include/freetype2/freetype/tttags.h
+/usr/local/include/freetype2/ft2build.h
```

Comment:
So if freetype stuff is here, why do we have a freetype package?

Reply:
The freetype is built against harfbuzz...

Reply:
no!